code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1173,6 +1173,15 @@
 			riding_datum.restore_position(user)
 	. = ..(user)
 
+/mob/living/silicon/robot/resist()
```

Comment:
could you make this only happen if the borg isn't trying to resist out of something else?

Reply:
yes if I gave made the resist verb use return values.

otherwise no.

obv being lazy here but the likelihood you're going to be in a situation where you need to unbuckle yourself as well as be on a timecrunch about putting the user back on will be very slim.